tutorials/image/cifar10/cifar10_input.py

```diff
@@ -84,13 +84,14 @@ class CIFAR10Record(object):
 
   # The first bytes represent the label, which we convert from uint8->int32.
   result.label = tf.cast(
-      tf.strided_slice(record_bytes, [0], [label_bytes]), tf.int32)
+      tf.strided_slice(record_bytes, [0], [label_bytes], strides=[1]), tf.int32)
 
   # The remaining bytes after the label represent the image, which we reshape
   # from [depth * height * width] to [depth, height, width].
   depth_major = tf.reshape(
       tf.strided_slice(record_bytes, [label_bytes],
-                       [label_bytes + image_bytes]),
+                       [label_bytes + image_bytes],
+                       strides=[1]),
```

Comment:
AFAIK The default value to `strides` in `tf.strided_slice` is `tf.ones_like([label_bytes])`. While they look same, I would suggest to avoid the use of magic number!